src/style.css

```diff
@@ -0,0 +1,133 @@
+html {
+  font-family: Roboto, sans-serif;
+  --main-text-color: #060b37;
+  --secondary-text-color: #616070;
+  --blue-accent: #00acdc;
+}
+
+body {
+  margin: 0;
+  /* box-sizing: border-box; */
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.product-card__img {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 40px;
+}
+
+.product-card__text {
+  padding: 0 16px;
+}
+
+.product-card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
```

Comment:
It's better to use a percentage or multiplier value for line height.
`lineheight: 1.5`